### PR TITLE
MBS-7669 Support draft pages in documentation preview

### PR DESCRIPTION
**Pull request by eugene-krivobokov**
Now it's easier to check draft pages:

https://gohugo.io/getting-started/usage/#draft-future-and-expired-content

```
---
title: ...
type: docs
draft: true
---
```

I hope to use drafts to merge some new pages before final polishing and publishing.

### CODE REVIEW

File: docs/local.sh [hunk 1]
```diff
@@ -28,9 +28,9 @@ openBrowser
 # Otherwise it'll crash hugo due to missing files.
 # You can find them in .dockerignore
 
-docker run -t --rm \
+docker run -t -i --rm \
```
Comment (eugene-krivobokov) [inline docs/local.sh:31]: to fix Ctrl + C handling

File: docs/content/docs/contributing/Docs.md [hunk 2]
```diff
@@ -294,27 +294,29 @@ Markdown content
 
 ### Lists
 
-#### Нумерованный или ненумерованный
+#### Ordered vs Unordered lists
 
-Используй нумерованный список только если он описывает последовательность шагов.
+Only use ordered list for describing an algorithm.
```
Comment (sergioboi) [inline docs/content/docs/contributing/Docs.md]: I need explanation.

File: docs/content/docs/contributing/Docs.md [hunk 3]
```diff
@@ -343,11 +345,14 @@ Open **Preferences > Build, Execution, Deployment > Debugger**
...
+### How to check changes locally
 
-Run: `docs/local.sh`\
+Run `docs/local.sh`\
 It will open documentation at `localhost:1313`.\
-You can edit markdown files in the `content/` on the fly. Changes out of this directory usually require a restart.  
+You can edit markdown files in the `content/` on the fly. Changes out of this directory usually require a restart.
+
+By default, we don't publish [draft, future and expired](https://gohugo.io/getting-started/usage/#draft-future-and-expired-content) pages.\
+To publish them, add relevant argument to the command: `docs/local.sh --buildDrafts`
```
Comment (sergioboi) [inline docs/content/docs/contributing/Docs.md:368]: Sorry. I don't understand why we need support that. Can you write about scenarios?

**Review by sergioboi — CHANGES_REQUESTED** [review]

File: docs/content/docs/contributing/Docs.md (hunk 2 above)
Comment (eugene-krivobokov) [reply docs/content/docs/contributing/Docs.md]: What about "a sequence of steps" or "step by step guide"? Is it ok?

File: docs/content/docs/contributing/Docs.md (hunk 3 above)
Comment (eugene-krivobokov) [reply docs/content/docs/contributing/Docs.md:368]: We don't add anything additional. It's a built-in feature that didn't work in this script.

My scenario:

- I want to add a new page but it's not completed yet: `draft: true`.
I've written it partially and don't want to show it to users.
- I commit this file as a draft. 
We discuss it in PR and have TODOs to further changes.
I can merge unfinished work.
- I do a final PR and "enable" this page to users.

Now, I need to create a google doc to discuss something unfinished. It's harder to check conflicts, we need a separate code review.

It's a complete analogy of feature-toggle but for a page. Do changes step by step, and publish explicitly.
I don't want to use it for all new pages. Only when we agree to merge it and fix it in the next PR.

File: docs/local.sh (hunk 1 above)
Comment (dsvoronin) [reply docs/local.sh:31]: i thought `-t` already fixed it;

Comment (eugene-krivobokov) [reply docs/local.sh:31]: Me too
https://docs.docker.com/engine/reference/run/#foreground

File: docs/content/docs/contributing/Docs.md (hunk 2 above)
Comment (eugene-krivobokov) [reply docs/content/docs/contributing/Docs.md]: @sboishtyan , ^

Comment (sergioboi) [reply docs/content/docs/contributing/Docs.md]: Explain why I need to use ordered list

Comment (eugene-krivobokov) [reply docs/content/docs/contributing/Docs.md]: Replaced by "series of steps to follow"

File: docs/content/docs/contributing/Docs.md [hunk 4]
```diff
@@ -307,27 +307,29 @@ Markdown content
 
 ### Lists
 
-#### Нумерованный или ненумерованный
+#### Ordered vs Unordered lists
 
-Используй нумерованный список только если он описывает последовательность шагов.
+Only use ordered list for describing a series of steps to follow.
```
Comment (int02h) [inline docs/content/docs/contributing/Docs.md:312]:
```suggestion
Use ordered list only for describing a series of steps to follow.
```

Comment (eugene-krivobokov) [reply docs/content/docs/contributing/Docs.md:312]: This is intentional. I focus on it.

https://dictionary.cambridge.org/grammar/british-grammar/only (Ordering)

**Review by sergioboi — APPROVED** [review]

**Pull request merged**